### ISSUE TITLE
Accessibility: Do not use the meta viewport tag to disable zoom

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-admin#1958]
[WCAG 2.2 AA 1.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html)
In the `templates/SilverStripe/Admin/LeftAndMain.ss` template, the `viewport` meta tag looks like this:

`<meta name="viewport" content="width=device-width, maximum-scale=1.0" />`

That prevents users from zooming on mobile devices. According to MDN any `maximum-scale` value less than 3 is an automatic fail for accessibility.

The `maximum-scale` should be removed, and most likely `initial-scale` should be added and set to `1.0`:

`<meta name="viewport" content="width=device-width, initial-scale=1.0" />`

See [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Guides/Viewport_meta_element) for more details about the viewport meta element and its options.

## Acceptance criteria

- [ ] `maximum-scale` is not used
- [ ] If appropriate, `initial-scale` is added

## PRs
- https://github.com/silverstripe/silverstripe-admin/pull/1993
- https://github.com/silverstripe/developer-docs/pull/810

**Comment by GuySartorelli** [comment]
PRs merged
